migrations/db/intermediate_db_schema/100-base-schema.sql

```diff
@@ -80,7 +80,7 @@ CREATE TABLE users
     approved                  BOOLEAN,
     approved_at               DATETIME,
     approved_by_id            NUMERIC,
-    created_at                DATETIME  NOT NULL,
+    created_at                DATETIME,
```

Comment:
I feel like we should require `created_at` for users. That's crucial information for users.